module/metadata.js

```diff
@@ -707,6 +707,41 @@ ARM5E.magic.durations = {
     dtype: "String",
     impact: 1
   },
+  recitation: {
+    label: "arm5e.spell.durations.recitation",
+    dtype: "String",
+    impact: 1
+  },
+  office: {
+    label: "arm5e.spell.durations.office",
+    dtype: "String",
+    impact: 1
+  },
+  storm: {
+    label: "arm5e.spell.durations.storm",
+    dtype: "String",
+    impact: 1
+  },
+  focus: {
+    label: "arm5e.spell.durations.focus",
+    dtype: "String",
+    impact: 1
+  },
+  minutes: {
+    label: "arm5e.spell.durations.minutes",
+    dtype: "String",
+    impact: 1
+  },
+  held: {
+    label: "arm5e.spell.durations.held",
+    dtype: "String",
+    impact: 1
+  },
+  while: {
+    label: "arm5e.spell.durations.while",
+    dtype: "String",
+    impact: 1
+  },
   sun: {
```

Comment:
I understand you want to order them by impact, but the core rules durations targets and ranges should be at the top as they are the most used (that's the order they will appear in the dropdown menu) Otherwise, they will be lost with all the others, of which only a handful will be used per Saga,